cpp/ql/test/library-tests/ir/ir/PrintAST.expected

```diff
@@ -5565,6 +5565,30 @@ ir.cpp:
 #  619|                 Type = [PointerType] const char *
 #  619|                 ValueCategory = prvalue
 #  620|     getStmt(4): [ReturnStmt] return ...
+#  620|       getImplicitDestructorCall(0): [DestructorCall] call to ~String
```

Comment:
This can be done in a separate PR, but I wonder if we should print these in the order in which they should be called, which is from the highest to the lowest index:

https://github.com/github/codeql/blob/32a2ea1690d0d9cee525b61757d2052279c39623/cpp/ql/lib/semmlecode.cpp.dbscheme#L1122-L1127

Reply:
They already are - I reversed the indexes from the dbscheme order in `getImplicitDestructorCall`.

Reply:
I see now. I was clearly not reading the code correctly 🤦 . Thanks for clarifying.